e2e_test/udf/sql_udf.slt

```diff
@@ -13,6 +13,13 @@ create function sub(INT, INT) returns int as 'select $1 - $2' language sql;
 statement ok
 create function add_sub_binding() returns int as 'select add(1, 1) + sub(2, 2)' language sql;
 
+# Create an anonymous function with return expression
+statement ok
+create function add_return(INT, INT) returns int return $1 + $2 language sql;
```

Comment:
Is this PG's syntax?

`returns int return` looks a little strange 🤣

Reply:
Seems not. Why do we want to support this?

Reply:
If we move `return` to the end, this statement can be accepted by pg.
```sql
create function add_return(INT, INT) returns int language sql return $1 + $2;
```

Reply:
seems so
```
CREATE FUNCTION name ( args ) RETURNS type
LANGUAGE SQL
    | AS 'definition'
    | sql_body
  } ...

sql_body := RETURN expression
```

Reply:
Yeah it's because I put `language sql` in the wrong place, and just for reference below is an `return` example from pg's document of `create function`:
```sql
CREATE FUNCTION add(a integer, b integer) RETURNS integer
    LANGUAGE SQL
    IMMUTABLE
    RETURNS NULL ON NULL INPUT
    RETURN a + b;
```